src/main.go

```diff
@@ -65,6 +65,11 @@ func main() {
 
 	fmt.Println("Bot is open and listening...")
 
+	dbHandler := infrastructure.NewSqliteHandler("/var/tmp/production.sqlite")
+
+	handlers := make(map[string]interfaces.DbHandler)
+	handlers["DbPlayerRepo"] = dbHandler
```

Comment:
If you plan to inject this into the delegator instantiate it in the init function like I have with the other repositories